keyboards/keycapsss/kimiko/keymaps/oriaj3/config.h

```diff
@@ -23,7 +23,7 @@
 // #define MASTER_RIGHT
 // #define EE_HANDS
 
-#define OLED_FONT_H "keyboards/keycapsss/kimiko/keymaps/rgb_matrix/glcdfont.c"
+#define OLED_FONT_H "keyboards/keycapsss/kimiko/keymaps/oriaj3/glcdfont.c"
```

Comment:
Can you revert this line, otherwise it will require user sign off or go via develop.

Reply:
I can, but then CI will fail (that was the only reason I changed it -- to get CI to pass, instead of failing on that non-extant file). Can you confirm that's what you would like me to do?

Reply:
CI failing on non-default keymaps is okay.